test/generic/NCPoly-test.jl

```diff
@@ -166,6 +166,11 @@ end
       @test f*d1 - f*d2 == f*(d1 - d2)
       @test f*d1 + f*d2 == f*(d1 + d2)
    end
+
+   # Fix for some ambiguity
+   QQx, x = QQ["x"]
+   y = gen(Generic.NCPolyRing{elem_type(QQx)}(QQx, :y, false))
+   @test y * x == x * y
```

Comment:
no bueno. With this patch, I get a method error for `y-2` here, which I didn't get before.